addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18615,7 +18615,13 @@ msgctxt "#36012"
 msgid "Couldn't initialise the CEC adapter. Please check your settings."
 msgstr ""
 
-#empty strings from id 36013 to 36014
+#. CEC adapter setting to set the type of device
```

Comment:
I feel bad leaving translators with little context, so this might be an improvement:

```
#. CEC adapter setting to set the type of device that Kodi reports to the TV (Recording Device, Playback Device or Tuner Device)
```


msgid "Recording Device"
msgstr ""

#: system/peripherals.xml
msgctxt "#36052"
msgid "Playback Device"
msgstr ""

#: system/peripherals.xml
msgctxt "#36053"
msgid "Tuner Device"